### DISCUSSION TITLE
Box Packer X Laravel

### DISCUSSION THREAD

**Discussion by arthuramorim** [dvdoug/BoxPacker#523] [Q&A]
Is "box packer" compatible with the current version of laravel?

**Comment by dvdoug** [comment]
BoxPacker is a standalone library, so framework version doesn't matter

**Comment by frankvw2017** [comment]
BoxPacker is essentially a collection of bricks and mortar to build your own implementation with. So in terms of "is it a plug-in that you can just drop into an off-the-shelf product" the answer would be no, but in terms of "can you build an implementation with it that will work within the Laravel PHP web application framework" then yes, definitely.
